Makefile.config.example

```diff
@@ -68,6 +68,10 @@ PYTHON_LIB := /usr/lib
 # Uncomment to support layers written in Python (will link against Python libs)
 # WITH_PYTHON_LAYER := 1
 
+# Uncomment to use Python 3 (default is Python 2)
+# PYTHON_LIBRARIES := boost_python3 python3.4m
+# Also, you'll need to edit PYTHON_INCLUDE above.
+
```

Comment:
One small suggestion: it might be worth moving these lines a bit up in the code, right below the anaconda example, so that we can also have a PYTHON_INCLUDE example and just comment out both PYTHON_LIBRARIES and PYTHON_INCLUDE if we switch to python3.